test/deps/leveldb_test.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <leveldb/db.h>
+#include <boost/filesystem.hpp>
+
+namespace fs = boost::filesystem;
+
+void clear(fs::path p) {
+  if (fs::exists(p)) {
+    fs::remove_all(p);
+  }
+}
+
+TEST(Leveldb, Integration) {
+  leveldb::Options options;
+  options.create_if_missing = true;
+
+  fs::path path = "/tmp/leveldb_deps_test";
+  clear(path);
+
+  fs::create_directory(path);
```

Comment:
directory is created but not removed. Better to move all initialization stuff to fixture and all clearings to TearDown

Reply:
removed this test